R/execution.r

```diff
@@ -63,6 +63,7 @@ Executor <- setRefClass(
     'Executor',
     fields = list(
         send_response      = 'function',
+        abort_queued_messages = 'function',
```

Comment:
reference classes are weird. they’re “OO like it works in java, added as an afterhought”